src/action/macos/create_nix_volume.rs

```diff
@@ -99,6 +103,15 @@ impl CreateNixVolume {
                 .map_err(Self::error)?;
         let enable_ownership = EnableOwnership::plan("/nix").await.map_err(Self::error)?;
 
+        let set_tmutil_exclusions = vec![
```

Comment:
Maybe worth adding a note that excluding `/nix` entirely did not work on our (managed) macs, so future code-combers will know why we did these 2 subdirs instead of their parent.